test/e2e/monitoring_stack_controller_test.go

```diff
@@ -861,7 +861,7 @@ func namespaceSelectorTest(t *testing.T) {
 
 	promClient := framework.NewPrometheusClient("http://localhost:9090")
 	if pollErr := wait.Poll(5*time.Second, 5*time.Minute, func() (bool, error) {
-		query := `prometheus_build_info{namespace=~"test-ns-.*"}`
+		query := `version{namespace=~"test-ns-.*"}`
```

Comment:
is there a different metric that we can reply on that is specific to `prometheus_example` ?

Reply:
Yes, there are the following metrics, for detail you can refer [web site](https://github.com/brancz/prometheus-example-app)
```
version - of type gauge - containing the app version - as a constant metric value 1 and label version, representing this app version
http_requests_total - of type counter - representing the total numbere of incoming HTTP requests
http_request_duration_seconds - of type histogram, representing duration of all HTTP requests
http_request_duration_seconds_count- total count of all incoming HTTP requeests
http_request_duration_seconds_sum - total duration in seconds of all incoming HTTP requests
http_request_duration_seconds_bucket
```

Reply:
I know what you mean, I changed the metric as `version{pod="prometheus-example-app",namespace=~"test-ns-.*"}`, thanks for the comments